### PR TITLE
Lvjav442 8

### PR DESCRIPTION
**Pull request by martaprotsak**
Implement media_type entity

### CODE REVIEW

File: src/main/java/com/softserve/ita/java442/cityDonut/model/MediaType.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+@Table(name = "media_type")
+public class MediaType {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "media_type", length = 5, nullable = false)
+    private String media_type;
+
+    @OneToMany(fetch = FetchType.EAGER, mappedBy = "mediaType")
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/MediaType.java]: fetchType.Lazy

File: src/main/java/com/softserve/ita/java442/cityDonut/model/MediaType.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "media_type")
+public class MediaType {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "media_type", length = 5, nullable = false)
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/MediaType.java:21]: dont know, but maybe exist media type with length more than 5?

File: src/main/java/com/softserve/ita/java442/cityDonut/model/MediaType.java (hunk 1 above)
Comment (martaprotsak) [reply src/main/java/com/softserve/ita/java442/cityDonut/model/MediaType.java]: In media entity FetchType is EAGER. Is this ok?

File: src/main/java/com/softserve/ita/java442/cityDonut/model/MediaType.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    private String media_type;
+
+    @OneToMany(fetch = FetchType.EAGER, mappedBy = "mediaType")
+    private List<MediaType> mediaTypeList;
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/MediaType.java]: 26 - private List generic(Media) media;

**Review by NazarRyvak — APPROVED** [review]

**Review by martavoloshyn — APPROVED** [review]

**Pull request merged**